.travis.yml

```diff
@@ -0,0 +1,13 @@
+language: node_js
+node_js: node
+after_success: 
+    - npm run build
+deploy:
+    provider: s3
+    access_key_id: "AKIAJGKZPVSGGK6DK4JA"
```

Comment:
You need to remove this, look up their convention for credentials, delete your role on aws, make a new one, and then update the PR.  If you leave it you might have people put a whole bunch of trash on s3 or use whatever services that role has permissions for.